examples/bodo/bodo-model-training.ipynb

```diff
@@ -0,0 +1,412 @@
+    "    rr = Ridge()\n",
+    "    rr_fitted = rr.fit(X_train, y_train)\n",
+    "    print(\"Ridge fitting time: \", time.time() - start)\n",
+    "\n",
+    "    start = time.time()\n",
+    "    rr_preds = rr_fitted.predict(X_test)\n",
+    "    print(\"Ridge prediction time: \", time.time() - start)\n",
+    "    print(mean_squared_error(y_test, rr_preds, squared=False))\n",
+    "\n",
+    "\n",
+    "rr_model(X_train, y_train, X_test, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "separated-session",
+   "metadata": {},
+   "source": [
+    "#### Lasso regression"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "executed-winning",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%px\n",
+    "@bodo.jit(distributed=[\"X_train\", \"y_train\", \"X_test\", \"y_test\"])\n",
+    "def lsr_model(X_train, y_train, X_test, y_test):\n",
+    "    start = time.time()\n",
+    "    lsr = Lasso()\n",
+    "    lsr_fitted = lsr.fit(X_train, y_train)\n",
+    "    print(\"Lasso fitting time: \", time.time() - start)\n",
+    "\n",
+    "    start = time.time()\n",
+    "    lsr_preds = lsr_fitted.predict(X_test)\n",
+    "    print(\"Lasso prediction time: \", time.time() - start)\n",
+    "    print(mean_squared_error(y_test, lsr_preds, squared=False))\n",
+    "\n",
+    "\n",
+    "lsr_model(X_train, y_train, X_test, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "greater-evolution",
+   "metadata": {},
+   "source": [
+    "#### SGD regressor model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "related-michael",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%px\n",
+    "@bodo.jit(distributed=[\"X_train\", \"y_train\", \"X_test\", \"y_test\"])\n",
+    "def sgdr_model(X_train, y_train, X_test, y_test):\n",
+    "    start = time.time()\n",
+    "    sgdr = SGDRegressor(max_iter=100, penalty=\"l2\")\n",
+    "    sgdr_fitted = sgdr.fit(X_train, y_train)\n",
+    "    print(\"SGDRegressor fitting time: \", time.time() - start)\n",
+    "\n",
+    "    start = time.time()\n",
+    "    sgdr_preds = sgdr_fitted.predict(X_test)\n",
+    "    print(\"SGDRegressor prediction time: \", time.time() - start)\n",
+    "    print(mean_squared_error(y_test, sgdr_preds, squared=False))\n",
+    "\n",
+    "\n",
+    "sgdr_model(X_train, y_train, X_test, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9413a73",
+   "metadata": {},
+   "source": [
+    "## Stopping the cluster\n",
+    "\n",
+    "When you're done using the parallel cluster you can shut it down with a single command. Note that since the cluster is running within the same Jupyter Server resource as the notebook, there is no change to what hardware is running after you stop it (since the resource is still one). Thus in this case there isn't a need to shut it off unless you wish to reset the cluster."
```

Comment:
I'm not sure what you mean by "(since the resource is still one)". Otherwise, this is useful if you are running multiple examples - it will spawn ever more processes if you don't shut down the "cluster"